.github/workflows/integration-tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         
         pip install -U pip setuptools
 
-        python setup.py install
+        pip install .
```

Comment:
how is this change related to the urllib3 problem?

Reply:
I thought it was the issue at first.

Since there was a ton of new warning that setup.py install is going to be deprecated.

I might move it to it's own commit